public/scripts/services/date.js

```diff
@@ -7,8 +7,7 @@
                 return null;
             }
 
-            return new Intl.DateTimeFormat('en-US', {
-                timeZone: 'UTC',
```

Comment:
This removal is intentional, when not included the browser's timezone will be used